package.json

```diff
@@ -602,6 +653,29 @@
           "type": "boolean",
           "default": false,
           "description": "Specifies whether notifications should be shown if OmniSharp encounters warnings or errors loading a project. Note that these warnings/errors are always emitted to the OmniSharp log"
+        },
+        "omnisharp.preview": {
+          "type": "boolean",
+          "default": false,
+          "description": "Specifies whether to enable OmniSharp preview features."
+        },
+        "razor.languageServer.path": {
```

Comment:
Since this is really pointing to the directory we should consider changing this to `razor.languageServer.directory`

Reply:
OK, done